packages/apfelxx/install.sh

```diff
@@ -0,0 +1,26 @@
+# early exit if errors occur
+set -e
+
+INSTALLER_DIR=$(realpath $(dirname $0))
+
+# import variables
+. $INSTALLER_DIR/variables.sh
+
+# download the code
+sh $INSTALLER_DIR/download.sh $URL
+
+# extract code
+cd $SRC_DIR
+rm -rf build
+mkdir build && cd build
+
+# if PREFIX is set use it, otherwise do the default
+if [ -z $PREFIX ]; then
+  cmake .. 
+else
+  echo "Installing in $PREFIX"
+  cmake -DCMAKE_INSTALL_PREFIX=$PREFIX ..
+fi
+
+# compile, and install (includes python interface)
+make && make install
```

Comment:
```suggestion
make -j && make install
```

You might want to add a `-j`, as we did for the other packages, to speed up compilation (when multiple cores are available)